modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldSettingUtil.java

```diff
@@ -89,11 +89,20 @@ public class ObjectFieldSettingUtil {
 		if (serviceBuilderObjectFieldSetting.compareName(
 				ObjectFieldSettingConstants.NAME_STATE_FLOW)) {
 
+			ObjectStateFlow objectStateFlow = null;
+
+			if (!(objectFieldSetting.getValue() instanceof ObjectStateFlow)) {
```

Comment:
In the case of this bug, the execution enter into the IF clause, right?
Do you know which case the execution enter in the ELSE?

Reply:
when it passes the second PUT of the import. objectDefinitionResource.putObjectDefinition(
putObjectDefinition.getId(), putObjectDefinition);